README.md

```diff
@@ -73,6 +73,9 @@ Per-VDOM:
  * _System/Fortimanager/Status_
    * `fortigate_fortimanager_connection_status`
    * `fortigate_fortimanager_registration_status`
+   * `fortigate_ippool_total_items`
```

Comment:
Copy paste error?

Reply:
yea, sorry, don't really know if to remove I have to recommit